packages/eventual-send/package.json

```diff
@@ -50,11 +50,12 @@
     "wavy dot"
   ],
   "files": [
+    "*.js",
+    "*.map",
```

Comment:
Are the generated files `*.map` as appears in these files or `*.map.js` as you said in the description?

Reply:
These are `*.ts.map`, TypeScript-generated source maps. Correcting the description.